addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -660,7 +660,7 @@ odoo.define('website.s_website_form', function (require) {
                 }
 
                 const formData = new FormData(this.$target[0]);
-                const currentValueOfDependency = formData.get(dependencyName);
+                const currentValueOfDependency = (comparator==="contains" || comparator==="!contains")?(formData.getAll(dependencyName).join())+"":formData.get(dependencyName);
```

Comment:
The line is now way too long and has linter errors, but also:
- Why the `+ ""`? Seems useless as if the `getAll` return an empty array, `.join()` will return already an empty string
- The set of `()` around `formData.getAll(dependencyName).join()` is useless
- Replace the double check by includes so you don't have to write twice `comparator` and should we add a third check it will be easier: `["contains", "!contains"].includes(comparator)`

All in all, just:
```suggestion
                const currentValueOfDependency = ["contains", "!contains"].includes(comparator)
                    ? formData.getAll(dependencyName).join()
                    : formData.get(dependencyName);
```

Reply:
I have made the changes